### PR TITLE
MTDSA-6647 RAML for empty body error, v1 and v2

### PR DESCRIPTION
**Pull request by yangwong9664**

### CODE REVIEW

File: resources/public/api/conf/1.0/docs/errors.md [hunk 1]
```diff
@@ -27,3 +27,12 @@ Where possible, multiple errors will be returned with `INVALID_REQUEST` in the f
         }
     ]
 }`
+
+Where it is possible for the same error to be returned multiple times, message will describe the expected format and paths will show the fields which are invalid.
+
+Where arrays are submitted a number indicates the object in the array sequence, for example, /arrayName/1/fieldName
+
+An example with single error:
```
Comment (james-work-account) [inline resources/public/api/conf/1.0/docs/errors.md:35]: Doesn't render properly - please copy from https://raw.githubusercontent.com/hmrc/individuals-income-received-api/master/resources/public/api/conf/1.0/docs/errors.md (note JSON indentation and backticks etc)

File: resources/public/api/conf/2.0/docs/errors.md [hunk 2]
```diff
@@ -27,3 +27,12 @@ Where possible, multiple errors will be returned with `INVALID_REQUEST` in the f
         }
     ]
 }`
+
+Where it is possible for the same error to be returned multiple times, message will describe the expected format and paths will show the fields which are invalid.
+
+Where arrays are submitted a number indicates the object in the array sequence, for example, /arrayName/1/fieldName
+
+An example with single error:
```
Comment (james-work-account) [inline resources/public/api/conf/2.0/docs/errors.md:35]: Doesn't render properly - please copy from https://raw.githubusercontent.com/hmrc/individuals-income-received-api/master/resources/public/api/conf/1.0/docs/errors.md (note JSON indentation and backticks etc)

Comment (yangwong9664) [reply resources/public/api/conf/2.0/docs/errors.md:35]: Ah when I copied it didn’t copy the back ticks, good spot, will add now

File: resources/public/api/conf/1.0/docs/errors.md (hunk 1 above)
Comment (yangwong9664) [reply resources/public/api/conf/1.0/docs/errors.md]: Ah when I copied it didn’t copy the back ticks, good spot, will add now

**Pull request merged**